actions/find_invite_MOD.js

```diff
@@ -43,12 +43,14 @@ module.exports = {
     const invite = this.evalMessage(data.invite, cache)
     const client = this.getDBM().Bot.bot
 
-    client.fetchInvite(invite).catch(console.error).then((invite) => {
-      const storage = parseInt(data.storage)
-      const varName = this.evalMessage(data.varName, cache)
-      this.storeValue(invite, storage, varName, cache)
-      this.callNextAction(cache)
-    })
+    client.fetchInvite(invite)
+      .catch(console.error)
+      .then((invite) => {
+        const storage = parseInt(data.storage)
+        const varName = this.evalMessage(data.varName, cache)
+        this.storeValue(invite, storage, varName, cache)
+        this.callNextAction(cache)
+      })
```

Comment:
If the `fetchInvite` call fails, the invite on the `then` callback would be `undefined` since that's what `console.error` returns. So unless that's what you wanted, change it to this.

```suggestion
      .then((invite) => {
        const storage = parseInt(data.storage)
        const varName = this.evalMessage(data.varName, cache)
        this.storeValue(invite, storage, varName, cache)
        this.callNextAction(cache)
      })
      .catch(console.error)
```